.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "6"
+  - "8"
```

Comment:
Is there a breaking change here that I need to be aware of?

Reply:
If I recall correctly, ember-macro-helpers has dropped support for Node 6 due to upgrading Ember CLI. Most addons consider this a breaking change.